site/_posts/2018-01-31-troubleshooting-auth.md

```diff
@@ -514,3 +515,42 @@ Look up groups for a specific user with ManageIQ by doing:
   1. Click *Retrieve*
   1. Near the top of the page a new drop-down should appear populated with group names the specified user is associated with on the IdP
 
+## Example Authentication Config
+---------------------------------------------------------------------
+
+This example, with screenshots, shows one possible authentication configuring.
```

Comment:
instead of configuring you probably meant: configuration?

Reply:
@abellotti Good catch! Thank you.